### ISSUE TITLE
CardboardSDKforUnity Import Failed in Unity 5.2.x

### ISSUE THREAD

**Issue by sbharathi** [googlesamples/io2015-codelabs#37]
In Unity 5.2.x, after importing the CardboardSDKforUnity, the Cardboard menu item is not getting created. It is also not working, when both 5.1.x and 5.2.x versions are installed on the system.

For now, quickly mention this in the codelab requirement. So that people won't try with latest Unity.

**Comment by claywilkinson** [comment]
Are you getting any errors on the import?  I just tried with Unity 5.2.0 and it imported OK.  Can you try clicking Assets/Refresh and if that does not work, Assets/ReImport All?

**Comment by sbharathi** [comment]
Did the Cardboard Menu Item got added in the menu list??

By the way, I am very new (few days) to Unity. For me, no error after import. Tried Refresh and Reimport All also.  But still cardboard menu is not visible.

**Comment by claywilkinson** [comment]
Yes, It seems to go away when you build.  But when I reimport all, it comes back.  Another workaround is to add the StereoController script to the main camera, then in the property inspector click, "Update Stereo Cameras"
